src/modules/component.js

```diff
@@ -1,6 +1,17 @@
-const component = () => {
-  const element = document.getElementById('test');
-  element.innerHTML = 'Welcome';
+const ul = document.getElementById('scoreBoard');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/bKuVOMomD7Dgr7nj8Sef/scores/';
+
+const loadScores = (data) => {
+  ul.innerHTML = '';
+  // eslint-disable-next-line array-callback-return
```

Comment:
- [ ] [OPTIONAL] It would be great if you avoid disabling eslint checks like this, so I recommend you to try to fix the issue instead. This is not like a big change you need to apply, but it is going to help you feel comfortable with the language and improve your thinking to solve issues. So keep it up!